lib/config/validation.ts

```diff
@@ -116,6 +118,15 @@ export async function validateConfig(
       }
     });
   }
+  if (!optionGlobals) {
+    optionGlobals = new Set<string>();
```

Comment:
```suggestion
    optionGlobals = new Set();
```
does typescript auto detect type?